codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/awssdk/DafnyAwsSdkClientTypeConversionProtocol.java

```diff
@@ -598,7 +600,7 @@ func Error_ToDafny(err error)($L.Error) {
             }
             """,
             DafnyNameResolver.dafnyTypesNamespace(serviceShape),
-            writer.consumer(w -> {
+              writer.consumer(w -> {
```

Comment:
Not sure how this came in diff but make format_java will fix this. So, Ignoring this for now.

Reply:
Personally I'm not a fan of our java formatter - it keeps line too short and I think uses 2 white-spaces for indentation.